glossary.md

```diff
@@ -276,6 +276,7 @@ Nếu bạn cho rằng một từ không nên dịch ra tiếng Việt, bạn c
 | one-hot encoding     | biễu diễn one-hot  | [https://git.io/JvohR](https://git.io/JvohR) |
 | one-sided test       | kiểm định một phía | [https://git.io/Jvoja](https://git.io/Jvoja) |
 | one-tailed test      | kiểm định một đuôi | [https://git.io/Jvoja](https://git.io/Jvoja) |
+| optimization landscape    | bối cảnh tối ưu  | [https://git.io/JvQxY](https://git.io/JvQxY) |
```

Comment:
```suggestion
| optimization landscape    | cảnh quan tối ưu  | [https://git.io/JvQxY](https://git.io/JvQxY) |
```

Reply:
có từ nào đỡ hơn từ "cảnh quan" không nhỉ?

"Toàn cảnh"?

Reply:
`toàn cảnh` em thấy gần với nghĩa `tổng quan` hơn. Em không nghĩ được từ gì khác nên em đồng ý với từ của @lkhphuc

Reply:
`toàn cảnh` thì chính xác với `global landscape` hơn.